rskj-core/src/test/java/org/ethereum/rpc/Web3ImplTest.java

```diff
@@ -1137,6 +1144,45 @@ public void lockAccount()
         org.junit.Assert.assertNull(account1);
     }
 
+    @Test
+    public void txpool_contentBasic() throws IOException {
+        World world = new World();
+        Web3Impl web3 = createWeb3(world);
+        String result = web3.txpool_content();
+        ObjectMapper om = new ObjectMapper();
+        JsonNode node = om.reader().forType(JsonNode.class).readValue(result);
```

Comment:
Maybe this won't scale with harder tests, but how about asserting the exact JSON string?

Reply:
These tests weren't pretending to test real values, just a basic syntactic structure.